build/dependencies.yaml

```diff
@@ -62,7 +62,7 @@ dependencies:
 
   # etcd
   - name: "etcd"
-    version: 3.5.9
+    version: 3.5.10
```

Comment:
Should we also be updating the etcd image on line 83 below?

Reply:
I just did that :)